### ISSUE TITLE
OSCAL model map revamp for M4 Metaschema

### ISSUE THREAD

**Issue by wendellpiez** [usnistgov/OSCAL#750]
# User Story:

With the Metaschema M4, models change. The model maps need to be updated.

See #725 on related documentation.

## Goals:

Prototype code with updated model maps can be found in the Metaschema M4 implementation. It needs to be integrated into Hugo and tested to produce maps for the new Metaschema design.

Also, @adamb-kpmg in Gitter (OSCAL channel Aug 11 16:11) reported a bug we can repair.

## Dependencies:

#721 and #724 must be complete for this work to proceed.

## Acceptance Criteria

- [x] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [x] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

**Comment by wendellpiez** [comment]
**Progress Sep 16**

I took a look and the outputs look good so far, but may require some adjustment when we integrate into Hugo.

In particular, we will need CSS something like this in order to support graceful collapse/expand using the HTML details element:

```
div { margin-left: 2em; }

details.OM-entry[open] > summary span.show-closed { display: none; }
```

Remaining effort on this is dependent on #721. Once we can see it fully styled in Hugo, we can touch up further as necessary.

**Comment by wendellpiez** [comment]
**Update Sep 24**

This is stable; next step as noted above is to integrate into the Hugo site production, then proof and correct once more if necessary.

**Comment by wendellpiez** [comment]
## Update Nov 5 2020

Stable pending Hugo integration work. Will discuss with @david-waltermire-nist.

**Comment by wendellpiez** [comment]
**Update Dec 3 2020**

Work continues; much progress. 90% done on revamp.

As with #725, I could use help from @david-waltermire-nist on git alignment.

**Comment by david-waltermire-nist** [comment]
This issue was addressed by PR #758.
